pages/docs/reference/delegated-properties.md

```diff
@@ -153,7 +153,45 @@ fun main() {
 If you want to intercept assignments and "veto" them, use [`vetoable()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.properties/-delegates/vetoable.html) instead of `observable()`.
 The handler passed to the `vetoable` is called _before_ the assignment of a new property value has been performed.
 
-## Storing Properties in a Map
+## Delegating to another property
```

Comment:
We can specify "(since 1.4)" as we did in the other headings here.

Reply:
Though I'd devise a label or note for that, rather than putting that in a heading.

Reply:
Yes, we're already getting rid of such notes in headings while moving to WebHelp.
We'll think up some kind of label for versions where they are important, and completely remove the old ones such as 1.1